components/src/dynamo/gpu_memory_service/args.py

```
@@ -0,0 +1,66 @@
+        "--verbose",
+        "-v",
+        action="store_true",
+        help="Enable verbose logging.",
+    )
+
+    args = parser.parse_args()
+
+    # Generate default socket path if not provided
+    socket_path = args.socket_path
+    if socket_path is None:
+        socket_path = f"/tmp/gpu_memory_service_{args.device}.sock"
```

Ruff (0.14.10)

Probable insecure usage of temporary file or directory: "/tmp/gpu_memory_service_"

(S108)